lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +140,18 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+        end
```

Comment:
I think maybe we should not list `--color` in `--help` since it is deprecated (but still support it).  We don't really want to take up valuable `--help` real estate with something deprecated.

OTOH, it is still supported, so I can see an argument for keeping it in `--help`.  Thoughts?  (Especially from @rspec/rspec people...)